evaluator.go

```diff
@@ -73,12 +78,24 @@ func (s *State) Clusters() map[string]Cluster {
 	return s.clusters
 }
 
+// enable clusterQueryResultCaching
```

Comment:
any way to quantify it?

My experience with RSpec has left me really reluctant to expand configuration API without really good reason, since it gets unruly really quick.